locale/ka/LC_MESSAGES/amo.po

```diff
@@ -1,4 +1,4 @@
-# 
```

Comment:
This seems like an unnecessary whitespace change.

Reply:
sure thanks